src/app/components/error-page/error-page.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-error-page',
+  templateUrl: './error-page.component.html',
+  styleUrls: ['./error-page.component.scss']
+})
+export class ErrorPageComponent implements OnInit {
+
+  constructor(private router: Router) { }
+
+  ngOnInit(): void {
+  }
+
+  goToHomePage() {
```

Comment:
Is this method needed?
Maybe it's better to use `routerLink` for the button?

Reply:
Fixed